src/lib/blockchain.js

```diff
@@ -47,7 +47,8 @@ export const getAccounts = () => {
 
 export const getTransactions = async () => {
   const maxTransactionsCount = 10;
-  const blocks = await getBlocks(10);
```

Comment:
We were previously taking the last 10 blocks... why did you decide to take only 3?
This change breaks existing functionality.